src/main/java/uk/gov/hmcts/reform/sscs/services/sftp/SftpChannelAdapter.java

```diff
@@ -82,48 +118,115 @@ public ChannelSftp getSftpChannel() {
         return channel;
     }
 
+    protected Session openSession() {
+        Session session = threadSession.get();
+        if (session == null) {
+            session = statelessConnect();
+            threadSession.set(session);
+        }
+        return session;
+    }
+
+    protected Session statelessConnect() {
+        Session session = null;
+        try {
+            initializeJch();
+            session = jsch.getSession(
+                    sftpSshProperties.getUsername(),
+                    sftpSshProperties.getHost(),
+                    sftpSshProperties.getPort());
+            session.setConfig("StrictHostKeyChecking", "no");
+            session.connect(60000);
+            return session;
+        } catch (JSchException e) {
+            throw new SftpCustomException("Session connection error", e);
+        }
+    }
+
     public List<Gaps2File> listIncoming() {
-        return list("");
+        return list(true,"");
     }
 
     public List<Gaps2File> listFailed() {
-        return list(FAILED_DIR);
+        return list(true, FAILED_DIR);
     }
 
     public List<Gaps2File> listProcessed() {
-        return list(PROCESSED_DIR);
+        return list(true, PROCESSED_DIR);
     }
 
     @SuppressWarnings("unchecked")
-    private List<Gaps2File> list(String path) {
-        ChannelSftp channel = getSftpChannel();
+    private List<Gaps2File> list(boolean closeSession, String path) {
```

Comment:
the closeSession argument is always true. Please, get rid of that unnecessary argument.